cypress/e2e/GUI/gui.cy.js

```diff
@@ -0,0 +1,141 @@
+/// <reference types="cypress" />
+describe('GUI Tests', () => {
+    beforeEach(function () {
+        cy.visit('/')
+        cy.contains('button', 'Accept').click()
+    })
+
+    it('Mostra as colunas Nome da Empresa e Acao, e oculta as colunas ID, Industria, Numero de Funcionarios e Tamanho em um viewport movel', () => {
+        cy.viewport(320, 480)
+        cy.contains('Company name').should('be.visible')
+        cy.contains('Action').should('be.visible')
+        cy.contains('Industry').should('not.be.visible')
+        cy.contains('Number of employees').should('not.be.visible')
+        cy.contains('Size').should('not.be.visible')
+    })
+
+    it('Exibe o cabecalho com um titulo, alternador de tema e um campo de entrada de texto', () => {
+        cy.contains('EngageSphere').should('be.visible')
+        cy.get('[class^="ThemeToggle_container"]').should('be.visible')
+        cy.get('input[id="name"]').should('be.visible')
```

Comment:
Um seletor mais simples seria:

```suggestion
        cy.get('#name').should('be.visible')
```

Se você pode usar uma solução mais simples, sempre dê preferência à ela.

Reply:
Isso torna o código mais curto e um pouco mais _clean_.